data/human/ships.txt

```diff
@@ -1260,6 +1260,61 @@ ship "Cruiser"
 
 
 
+ship "Cutthroat"
+	sprite "ship/pirate cutthroat"
+	thumbnail "thumbnail/pirate cutthroat"
+	attributes
+		category "Light Warship"
+		"cost" 1100000
+		"shields" 2800
+		"hull" 600
+		"required crew" 4
+		"bunks" 6
+		"mass" 171
+		"drag" 3
+		"heat dissipation" .8
+		"fuel capacity" 500
+		"cargo space" 37
```

Comment:
```suggestion
		"cargo space" 27
```
Need to take this down even more considering it got 31 more engine and 22 more outfit.